pkg/planner/cardinality/selectivity_test.go

```diff
@@ -1205,7 +1205,7 @@ func TestIgnoreRealtimeStats(t *testing.T) {
 	testKit := testkit.NewTestKit(t, store)
 	testKit.MustExec("use test")
 	testKit.MustExec("drop table if exists t")
-	testKit.MustExec("create table t(a int, b int, index ib(b))")
```

Comment:
May I ask why we need to remove this index?

Reply:
These tests are to test the realtime stats values. The presence of the index can result in the plan flipping to use an index for some of the tests - since stats are pseudo. Since the test is to demonstrate the realtime stats value and not to test the plan choice, I removed the index so that the plans do not change. Otherwise we need an "ignore index" on the queries to ensure that we keep a tablescan so that the actual realtime stats values can be observed for the test.